queries/futures/types.ts

```diff
@@ -163,9 +163,11 @@ export type FuturesTradeWithPrice = {
 };
 
 export type FuturesTrade = {
-	size: string;
+	size: Wei;
 	asset: string;
-	price?: string;
+	price?: Wei;
+	trxId: string;
```

Comment:
Is `trxId` correct? Other places we use `txnHash`

Reply:
well, it's `id` in query so I just use `trxId`. Will change it to `trxHash`.